documentation/docs/10-getting-started/30-project-structure.md

```diff
@@ -4,27 +4,31 @@ title: Project structure
 
 A typical SvelteKit project looks like this:
 
+> Directories surrounded in brackets `[]` indicate they are optional and can be
+> added after starting a project.
```

Comment:
```suggestion
> Directories surrounded in brackets `[]` indicate they are optional and can be added after starting a project.
```

Reply:
Ah, my prettier prosewrap config got in the way here; will fix.